packages/transition-frontend/webpack.config.js

```diff
@@ -53,9 +53,13 @@ module.exports = (env) => {
   ];
 
   return {
+    stats: {
```

Comment:
Comments for this?

Reply:
No clue! But ok, I,ll try to figure it out.... *grumble grumble*

Reply:
I guess it's that: https://webpack.js.org/configuration/stats/   ??